src/transformers/modeling_tf_bert.py

```diff
@@ -518,7 +518,7 @@ def __init__(self, config, **kwargs):
         self.return_dict = config.use_return_dict
         self.embeddings = TFBertEmbeddings(config, name="embeddings")
         self.encoder = TFBertEncoder(config, name="encoder")
-        self.pooler = TFBertPooler(config, name="pooler")
+        self.pooler = TFBertPooler(config, name="pooler") if add_pooling_layer else None
```

Comment:
Thanks a lot for adding this - it's great that it works now :-) ! Could you maybe also add it for TFAlbert, TFMobileNet and TFLongformer as is done in the PyTorch versions?

Reply:
Ok!